### PR TITLE
[wix-ui-core] Introduce tree-shakable stylable entry files

### PR DESCRIPTION
**Pull request by mykas**

### CODE REVIEW

File: packages/wix-ui-core/hocs.es.st.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
 
+:import {-st-from: "./dist/es/src/hocs/hocs.st.css";-st-default:Hocs;}
```
Comment (mykas) [inline packages/wix-ui-core/hocs.es.st.css:2]: This just got generated automatically :/

File: packages/wix-ui-core/package.json [hunk 2]
```diff
@@ -8,6 +8,10 @@
   "stylable": {
     "manifest": "./dist/standalone/src/stylable.manifest.json"
   },
+  "browser": {
+    "./index.st.css": "./dist/es/src/components/index.st.css",
```
Comment (mykas) [inline packages/wix-ui-core/package.json:12]: This is the magic we all know that should in theory allow us dropping those nasty script thats search & replace

File: packages/wix-ui-core/scripts/patch-stylable/tasks/transpile-patch-export.js [hunk 3]
```diff
@@ -34,19 +39,29 @@ module.exports = function({folder, entryFile}) {
...
   );
 
-  if(entryFile) {
+  if (entryFile) {
     entryFileCJS = execa.command(
       `stc --srcDir=./dist/src/${folder} --indexFile=${entryFile}.st.css`,
     );
 
-   entryFileES = execa.command(
+    entryFileESinES = execa.command(
```
Comment (mykas) [inline packages/wix-ui-core/scripts/patch-stylable/tasks/transpile-patch-export.js:47]: thats the only change - other things around are just lint

File: packages/wix-ui-core/package.json [hunk 4]
```diff
@@ -8,6 +8,10 @@
   "stylable": {
     "manifest": "./dist/standalone/src/stylable.manifest.json"
   },
+  "browser": {
+    "./index.st.css": "./index.es.st.css",
```
Comment (mykas) [inline packages/wix-ui-core/package.json:12]: this is the magic that makes it full tree-skakeable

**Review by EgoziE — APPROVED** [review]
Sure why not

**Comment by mykas** [comment]
@jonathanadler said no

**Comment by jonathanadler** [comment]
Computer says no

**Comment by jonathanadler** [comment]
Sorry guys, I'm closing this PR.
If we want to continue with this, let's have a conversation with Liron and Nitay

**Pull request closed without merging**